lib/Elevate/Blockers/Databases.pm

```diff
@@ -111,9 +111,31 @@ sub _blocker_old_mysql ($self) {
     return $mysql_is_provided_by_cloudlinux ? $self->_blocker_old_cloudlinux_mysql() : $self->_blocker_old_cpanel_mysql();
 }
 
-# TODO: RE-234 implement this
 sub _blocker_old_cloudlinux_mysql ($self) {
-    return;
+    my $cloudlinux_database_info = cpev::read_stage_file('cloudlinux_database_info');
```

Comment:
what if this comes back as undef?

Reply:
wouldn't it be better to call Elevate::DB::xxx to get the answer and leave the caching to that sub?

Reply:
It would be better.  Updated.